cmd/kube-apiserver/app/preflight/checks.go

```diff
@@ -21,67 +21,46 @@ import (
 	"net"
 	"net/url"
 	"time"
-
-	utilwait "k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/kubernetes/cmd/kube-apiserver/app/options"
 )
 
-const retryLimit = 60
-const retryInterval = 1 * time.Second
+const RetryLimit = 60
```

Comment:
Is there a reason for these constants to be public?  They're not tied to this package, and don't seem to provide much value for others.